app/controllers/boards_controller.rb

```diff
@@ -10,6 +10,9 @@ def index
 
   # GET /boards/1 or /boards/1.json
   def show
+    @mines = Mine.where(board_id: @board.id).order(height: :asc, width: :asc)
+    @tile = "\u2B1C ".freeze
+    @bomb = "💣 ".freeze
```

Comment:
The freeze methods tell ruby this wont change, which enables performance optimization:
```

3.0.0 :008 > Benchmark.measure { "hello".freeze * 1_000_000 }
 => #<Benchmark::Tms:0x00007fd46c8d6420 @label="", @real=0.0038420000000769505, @cstime=0.0, @cutime=0.0, @stime=0.0025400000000000422, @utime=0.0009019999999999584, @total=0.0034420000000000006> 
 
3.0.0 :009 > Benchmark.measure { "hello" * 1_000_000 }
 => #<Benchmark::Tms:0x00007fd46d0cbf90 @label="", @real=0.10360900000068796, @cstime=0.0, @cutime=0.0, @stime=0.001796999999999993, @utime=0.10176399999999974, @total=0.10356099999999974>
```